web/components/contract/quick-bet.tsx

```diff
@@ -233,9 +268,6 @@ export function getColor(contract: Contract) {
   }
 
   const marketClosed = (contract.closeTime || Infinity) < Date.now()
-  return marketClosed
-    ? 'gray-400'
-    : getProb(contract) >= 0.5
-    ? 'primary'
-    : 'red-400'
+  const prob = previewProb ?? getProb(contract)
+  return marketClosed ? 'gray-400' : prob >= 0.5 ? 'primary' : 'red-400'
```

Comment:
I think most designs that involve red + green together aren't that good because it's too many colors on a single item (plus the indigo of the question text). Though if you mock up any and think they look good, please send them my way!